css/details.css

```diff
@@ -12,13 +24,143 @@ th {
 }
 
 td {
-	text-align: right;
+	text-align: left;
+	padding-left: 1em;
+	max-width: 4em;
 }
 
 summary {
 	cursor: pointer;
+	margin-bottom: 1em;
+	user-select: none;
+}
+
+/* Imprecise rows */
+
+tr.imprecise {
+	font-style: italic;
+	z-index: 10;
+}
+
+tr.imprecise td {
+	position: relative;
+}
+
+/* Question mark circle icon */
+tr.imprecise td::after {
+	content: "?";
+	position: absolute;
+	top: 0.65em;
+	right: -1.2em;
+	width: 1.2em;
+	height: 1.2em;
+	border-radius: 1em;
+	border: 1px solid #333;
+	font-size: 90%;
+	font-style: normal;
+	color: #000;
```

Comment:
No, this is a really good way to do it! Sorry, I was just talking about the `color` property. By setting it here, we risk having a widget that fits the color chart of the host page, save for the question mark itself. I was curious if it was intentional.